version.json

```diff
@@ -1,17 +1,17 @@
 {
-    "main": {
-        "protoc_version": "22-dev",
+    "22.x": {
```

Comment:
I don't think there should be any changes to this file

Reply:
haha yep, fixed already (this was for testing)